data/assets/util/asset_helper.asset

```diff
@@ -82,35 +83,21 @@ local registerDashboardItems = function (dashboardAsset, items)
 end
 
 local registerSceneGraphNodesAndExport = function (sceneAsset, nodes, override)
-  override = override or false
-  if not override then
-    if tableLength(nodes) == 0 then
-      openspace.printWarning(sceneAsset.filePath .. ": Register function was called with an empty node list. Pass 'true' as third argument to silence this warning.")
-      return
-    end
-  end
+  registerSceneGraphNodes(sceneAsset, nodes, override)
 
   for i, node in ipairs(nodes) do
-    if not node.Identifier then
-      openspace.printError("Could not load asset as Identifier was missing")
-    end
+    sceneAsset.export(node.Identifier, node)
   end
+end
 
-  sceneAsset.onInitialize(function ()
-    for i, node in ipairs(nodes) do
-      openspace.addSceneGraphNode(node)
-    end
-  end)
-  sceneAsset.onDeinitialize(function ()
-    for i = #nodes, 1, -1 do
-      local node = nodes[i]
-      openspace.removeSceneGraphNode(node.Identifier)
-    end
+local registerLayerAndExport = function (sceneAsset, layer, globeIdentifier, layerGroup)
+
+  sceneAsset.onInitialize(function () 
+    openspace.globebrowsing.addLayer(globeIdentifier, layerGroup, layer)
+    sceneAsset.registerIdentifierWithMeta(layer.Identifier)
```

Comment:
This should also define the `onDeinitialize` method to remove the layers when the asset gets unloaded